eks/eks.go

```diff
@@ -1548,3 +1548,15 @@ func catchInterrupt(lg *zap.Logger, stopc chan struct{}, once *sync.Once, sigc c
 	}
 	return err
 }
+
+func (ts *Tester) ScaleMNG() {
```

Comment:
We don't need call this here, how about pattern like this?

https://github.com/aws/aws-k8s-tester/blob/337486418c719b4bce1491329b4b08a4011e4185/eks/eks.go#L1199-L1228